### ISSUE TITLE
How do I compile on Linux?

### ISSUE THREAD

**Issue by ConLenov** [dsfb/JavaQemu#8]
I use Ubuntu. How do I compile java qemu for Ubuntu?

**Comment by dsfb** [comment]
Hi, @ConLenov!

I can make a Ubuntu package for this software JavaQemu, that is, a deb format file, of course!

I will put this issue in my TODO task list!

Thanks!

Best Regards,

**Comment by ConLenov** [comment]
The thing is that I want to make a javaqemu translation to russian. Do you allow me to make a translation?

**Comment by dsfb** [comment]
@ConLenov Yes, you can make a JavaQemu translation to Russian Language, of course!
Thanks!

**Comment by dsfb** [comment]
@ConLenov I will create some separated files with the sentences used in JavaQemu Software, in another issue!
Then, you could to translate the English Language file for the Russian Language!
Thanks!

**Comment by dsfb** [comment]
#9

**Comment by dsfb** [comment]
@ConLenov If you still want to compile this project, use the Eclipse Java SE IDE!
This IDE can be obtained from the site:
[Homepage for the Eclipse IDEs](https://www.eclipse.org/downloads/)
.

Thanks!
